config/locales/candidate_interface/other_qualification.yml

```
@@ -73,6 +73,6 @@ en:
               too_long: The grade must be %{count} characters or fewer
             award_year:
               blank: Enter the year the qualification was awarded
-              future: Assessment year must be this year or a previous year
+              future: Assessment year must be next year, this year or a previous year
```

Update to match label wording ("awarded" instead of "assessed") and de-emphasise the "next year" option.

```suggestion
              future: Year qualification awarded must be this year, next year or a previous year
```

---

done